sacrebleu/sacrebleu.py

```diff
@@ -80,8 +80,11 @@ def parse_args():
                             help='Use a subset of sentences whose document annotation matches a given regex (see SUBSETS in the source code).')
     arg_parser.add_argument('--download', type=str, default=None,
                             help='Download a test set and quit.')
-    arg_parser.add_argument('--echo', choices=['src', 'ref', 'both'], type=str, default=None,
-                            help='Output the source (src), reference (ref), or both (both, pasted) to STDOUT and quit.')
+    arg_parser.add_argument('--echo', nargs="*", type=str, default=None,
+                            help='Output the source (src), reference (ref), or other avaliable field (docid, ref:A, ref:1 for example) to STDOUT and quit. '
+                                 'You can get avaliable fields with options `--list` and `-t`' 'For example: `sacrebleu -t wmt21 --list`. '
+                                 'If multiple fields are given, they are outputted with tsv format in the order they are given.'
+                                 'You can also use `--echo all` to output all avaliable fields.')
 
```

Comment:
`avaliable -> available` in three places (line 84, 85, 86)

Reply:
Fixed.